src/lib/serviceRoutines/deleteIndividualContextEntity.cpp

```diff
@@ -90,7 +93,10 @@ std::string deleteIndividualContextEntity
   }
 
   // 05. Cleanup and return result
+  TIME_STAT_RENDER_START();
```

Comment:
In order to avoid duplication (we have this same pattern in many places now), could a macro help to "wrap" the _START() and _STOP()? Something like this (probably not fully correct from a syntax point of view):

```
#define renderWithTimeStat(s, object, function, params)
TIME_STAT_RENDER_START();
s = object.function(params);
TIME_STAT_RENDER_STOP();
```

To be use like this:

```
renderWithTimeStat(answer, response, render, (ciP->outFormat, "", false, false))
```

(If render function is always the same, with name `render`, the macro definition could be simplified to `#define renderWithTimeStat(s, object, function, params)`)